dependency-suppression/cve-suppressed.xml

```diff
@@ -46,6 +46,7 @@
         <cve>CVE-2022-1471</cve>
         <cve>CVE-2022-38752</cve>
         <cve>CVE-2022-41854</cve>
+        <cve>CVE-2023-37475</cve>
```

Comment:
For Reference: https://github.com/din-global/schedule/actions/runs/5723787793/job/15509007910?pr=17